zerver/tornado/event_queue.py

```diff
@@ -1193,7 +1201,7 @@ def process_message_update_event(
                 pm_mention_email_disabled_user_ids=pm_mention_email_disabled_user_ids,
                 stream_push_user_ids=stream_push_user_ids,
                 stream_email_user_ids=stream_email_user_ids,
-                wildcard_mention_user_ids=wildcard_mention_user_ids,
+                stream_wildcard_mention_user_ids=stream_wildcard_mention_user_ids,
```

Comment:
This module needs to support the old format, because there can be events containing a `wildcard_mention_user_ids` field in the Tornado queues at the time the Zulip server is upgraded, and we need those to not break. Usually it's pretty simple -- if a field by that name exists, replace it with a `stream_wildcard_mention_user_ids` field. 

Search for `TODO/compatibility` in this file for examples of how to do this.

There may be a similar issue for the output of `maybe_enqueue_notifications`, too -- we should either avoid changing the format of what is enqueued by that function, or add similar compatibility code.

Reply:
As suggested in [#25828 (comment)](https://github.com/zulip/zulip/pull/25828#issuecomment-1581675528) -- addressed this in #25978